NextcloudTalk/UserStatusMessageSwiftUIView.swift

```diff
@@ -0,0 +1,369 @@
+//
+// Copyright (c) 2023 Lukas Lauerer <lukas.lauerer@gmx.net>
+//
+// Author Lukas Lauerer <lukas.lauerer@gmx.net>
+//
+// GNU GPL version 3 or any later version
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+import NextcloudKit
+
+struct UserStatusMessageSwiftUIView: View {
+
+    @Environment(\.dismiss) var dismiss
+    @Binding var changed: Bool
+    @State var showClearAtAlert: Bool = false
+
+    @State private var changedStatusFromPredif: Bool = false
```

Comment:
```suggestion
    @State private var changedStatusFromPredefined: Bool = false
```

?
Or would something like "predefinedStatusSelected" make more sense here?